src/docs/user/references/SchedulerGraphQLAPI.adoc

```diff
@@ -0,0 +1,380 @@
+
+====  Scheduler GraphQL API portal
+
+ProActive Scheduler GraphQL API portal can be accessed in your environment using the following link:
+
+++++
+<a class="schedulerGraphQLUrl" href="/scheduling-api/v1/graphiql" target="_blank">Scheduler GraphQL API</a> 
+++++
+
+In general the GraphiQL interface can be reached with `http://your-server:8080/scheduling-api/v1/graphiql`
+
+In order to use it, you must be first connected to one of the ProActive portals. If you are not currently connected or if your session has expired, you will receive a `MissingSessionIdException` when trying to run queries.
+
+This portal allows to test and run GraphQL queries against the scheduler database, to find jobs according to various filters.
+GraphQL queries are read-only, you cannot use this portal to perform stateful operations, such as submitting a job, deleting a job, etc.
+
+The GraphiQL interface is displayed as the following:
+
+image::SchedGraphqlMain.png[align="center"]
+
+You can explore the `Docs` section to see detailed information about all graphql nodes, filters and attributes.
```

Comment:
link to docs ?

Reply:
there is no link to Docs, this is the Docs section present in the graphiql portal. The user can expand this section to see the full documentation. I will add a corresponding screenshot.